doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,187 @@
+# Mount a Repo to a Local Computer
+
+Sometimes you might need to access files directly in
+a Pachyderm repository without downloading them first
+to your local computer. Pachyderm enables
+you to do so with the `pachctl mount` command. This command
+uses the SSH Filesystem (SSHFS) client and Filesystem
+in Userspace (FUSE) user interface to export a Pachyderm
+File System (PFS) to a Unix computer system. In other words,
+it allows you to mount a Pachyderm repository on your computer.
+
+For example, you have an [OpenCV pipeline](../../../getting_started/beginner_tutorial/#image-processing-with-opencv)
+up and running,
+and you need to edit files in the `images` repository. Let's
+say you want to experiment with brightness and contrast
+settings in `liberty.png`. Then, you want the pipelines
+run for the updated file and see the result in the end.
+If you do not mount the `images` repo, you would have to
+first download the files to your computer, edit them,
+and then put them back to the repository. The `pachctl mount`
+command automates all these steps for you. You can mount just the
```

Comment:
None of this is technically incorrect, but it should come way later once we introduce the idea of the writable mount. I think once we re-order things a bit and turn it into the clearer narrative, we can come back and I can help edit this paragraph further although it probably doesnt need much work.